deploy/versionChangelog.mjs

```diff
@@ -0,0 +1,37 @@
+// @ts-check
+
+// npm run ts-changelog @types/web 0.0.1 0.0.3
+
+import { generateChangelogFrom } from "../lib/changelog.js";
+import fetch from "node-fetch";
+
+const [name, before, to] = process.argv.slice(2);
+if (!name || !before || !to) {
+  throw new Error(
+    "Expected three arguments: package name, version before, version to"
+  );
+}
+
+const go = async () => {
+  const allFiles = `https://unpkg.com/${name}/?meta`;
+  const npmFileReq = await fetch(allFiles);
+  const npmDTSFiles = await npmFileReq.json();
+  const dtsFiles = npmDTSFiles.files.filter((f) => f.path.endsWith(".d.ts"));
+
+  for (const file of dtsFiles) {
+    const beforeURI = `https://unpkg.com/${name}@${before}${file.path}`;
```

Comment:
Okay this is a new code and I think it can use git tags. The required tools are already in `version.ts`.